src/postamble.js

```diff
@@ -43,6 +43,9 @@ if (memoryInitializer) {
     memoryInitializer = Module['locateFile'](memoryInitializer);
   } else if (Module['memoryInitializerPrefixURL']) {
     memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
+  } else if (ENVIRONMENT_IS_NODE) {
+    // Look for .mem file in the same folder as script by default
+    memoryInitializer = __dirname + '/' + memoryInitializer;
```

Comment:
Some questions do come to mind:
 - does node.js have a path join function that could be used here? I.e. is it guaranteed that `__dirname` does not end in a `'/'` so that this won't do a `/path/to/foo//foo.mem`? On Windows, should this be a backslash `'\'` or is node.js satisfied or even expect `'/'` on Windows as well?
 - are other file loads with node.js affected similarly? e.g. why is `.mem` file special here for node.js?

Last, I wonder if this would be better to read

    if (typeof Module['locateFile'] === 'function') {
      memoryInitializer = Module['locateFile'](memoryInitializer);
    } else if (Module['memoryInitializerPrefixURL']) {
      memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
    }

    + if (ENVIRONMENT_IS_NODE && isThisARelativeUrl(memoryInitializer)) {
    +   // Relative paths should be treated relative to the folder where the .js script resides in
    +   memoryInitializer = __dirname + '/' + memoryInitializer;
    + }

since `Module['locateFile']` and `Module['memoryInitializerPrefixURL']` may also be relative - do we ever want to load files relative to the cwd in node.js?